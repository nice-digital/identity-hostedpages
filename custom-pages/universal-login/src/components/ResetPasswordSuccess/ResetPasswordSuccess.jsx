import { Alert } from "@nice-digital/nds-alert";

export const ResetPasswordSuccess = () => {
  // setTimeout(() => (document.location = '/login'), 5000)
  return (
    <div>
      <h2>Thank you</h2>
      <Alert type="success">
        Your password has been changed.
      </Alert>
    </div>
  )
}
