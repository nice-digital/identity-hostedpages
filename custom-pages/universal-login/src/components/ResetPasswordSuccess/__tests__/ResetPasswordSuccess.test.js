import { render } from "@testing-library/react";
import { ResetPasswordSuccess } from "../ResetPasswordSuccess";

describe("ResetPasswordSuccess components", () => {
	it("should render ResetPasswordSuccess correctly", () => {
		const { container } = render(<ResetPasswordSuccess />);
		expect(container).toMatchSnapshot();
	});
});
