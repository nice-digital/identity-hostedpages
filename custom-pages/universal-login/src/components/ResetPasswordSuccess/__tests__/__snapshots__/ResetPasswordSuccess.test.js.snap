// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordSuccess components should render ResetPasswordSuccess correctly 1`] = `
<div>
  <div>
    <h2>
      Thank you
    </h2>
    <div
      class="alert alert--success"
      data-component="alert--success"
    >
      Your password has been changed.
    </div>
  </div>
</div>
`;
